CMakeLists.txt

```diff
@@ -64,6 +64,14 @@ target_compile_features(${POCLIB}
     cxx_std_14
 )
 
+#NOTE this option is done downstream, maybe poc should control it
+set(with_cxx "$<COMPILE_LANGUAGE:CXX>")
+set(ps_kokkos "$<BOOL:${PORTABILITY_STRATEGY_KOKKOS}>")
+
+target_compile_options(${POCLIB}
+INTERFACE
+$<${with_cxx}:$<${ps_kokkos}:--expt-relaxed-constexpr>>)
```

Comment:
In the event of +kokkos~cuda, wouldn't this result in a compile error?

Reply:
Looks like it, but POC doesn't have a `USE_CUDA` or similar like `singularity-eos` does, so I need to probe which compiler is being used.

Reply:
This is how I've done it in other projects:

```cmake
  get_target_property(kokkos_link_libs Kokkos::kokkoscore
                      INTERFACE_LINK_LIBRARIES)
  string(REGEX MATCH "CUDA"
         kokkos_has_cuda_libs "${kokkos_link_libs}")
  ...
  if(kokkos_has_cuda_libs)
    do cuda stuff...
  endif()
```